api/v1alpha1/groupversion_info.go

```diff
@@ -26,7 +26,8 @@ import (
 
 var (
 	// GroupVersion is group version used to register these objects
-	GroupVersion = schema.GroupVersion{Group: "operator.openshift.io", Version: "v1alpha1"}
+	// TODO: settle on an appropriate group name.
+	GroupVersion = schema.GroupVersion{Group: "externaldns.olm.openshift.io", Version: "v1alpha1"}
```

Comment:
this looks ok to me.  Check with @ecordell to see what's standard here.  operator.openshift.io is managed in openshift/api, so we want a unique group since this will not be coupled to an OCP payload and will update at a different cadence.

We can also carve off a subdomain under operator.openshfit.io if necessary, but your likely names may impact your future API choices that you *do* want coupled to OCP release cadence.

Reply:
I see the SR-IOV operator  (also installed via OLM) uses `sriovnetwork.openshift.io`. Could we jump a level up here and use `externaldns.openshift.io`?
Thoughts @ecordell ? TIA